project/src/main/scala/licenses/report/Review.scala

```diff
@@ -105,29 +105,35 @@ case class Review(root: File, dependencySummary: DependencySummary) {
 
     val unexpectedConfigurations =
       foundConfigurations.filter(p => !expectedFileNames.contains(p.getName))
-    val diagnostics = unexpectedConfigurations.map { p =>
+    val diagnostics = unexpectedConfigurations.map { p: File =>
       val packageNameFromConfig = p.getName
       val matchingPackages = knownPackages.filter(other =>
         packageNameFromConfig.startsWith(other.packageNameWithoutVersion + "-")
       )
       val maybeMatchingPackage: Option[DependencyInformation] =
         if (matchingPackages.length == 1) Some(matchingPackages.head) else None
 
-      val commonMessagePrefix =
-        s"Found legal review configuration for package ${p.getName}, but no such dependency has been found."
       maybeMatchingPackage match {
         case Some(matchingPackage) =>
           Diagnostic.Error(
-            commonMessagePrefix + s" Perhaps the version was changed to `${matchingPackage.packageName}`?",
+            s"Found legal review configuration for package ${p.getName}, but " +
+            s"no such dependency has been found. Perhaps the version was " +
+            s"changed to `${matchingPackage.packageName}`?",
             metadata = Map(
               "class"     -> "rename-dependency-config",
               "data-from" -> packageNameFromConfig,
               "data-to"   -> matchingPackage.packageName
             )
           )
         case None =>
-          Diagnostic.Error(
-            commonMessagePrefix + " Perhaps it has been removed or renamed (version change)?"
+          // The configuration is not related to any known package, so we remove it
+          IO.delete(p)
+          Diagnostic.Warning(
+            s"Found legal review configuration for package ${p.getName}, but " +
+            s"no such dependency has been found. It seems that the " +
+            s"dependency has been removed, so the configuration has been " +
+            s"deleted. If you think this was mistake, please rely on " +
+            s"version control to bring it back."
```

Comment:
@JaroslavTulach per your request - if I could not match a bumped version which could be renamed from the GUI, we assume the dependency was removed and _automatically_ delete the obsolete directory.
The warning is emitted to let the user know but it's a Warning not Error so it does not fail the report check.